packages/cli/src/index.ts

```diff
@@ -162,10 +165,15 @@ configProgram
     .description(
         'Interactively choose project.\nSee https://docs.lightdash.com/guides/cli/cli-authentication#set-active-project for more help and examples',
     )
-    .description('Interactively choose project')
     .option('--verbose', undefined, false)
-
     .action(setProjectInteractivelyHandler);
+configProgram
+    .command('set-project-by-name <projectName>')
```

Comment:
would be nice to have something like `set-project --uuid=uuid`

Reply:
Yes... maybe best would be to pass _either_ `--uuid` or `--name` as an option. I initially choose this approach of adding a new subcommand because I wanted to extend, not modify. However, I like the two options as alternatives. Giving the most flexibility.